connectors/katalog/pkg/connector/connector.go

```diff
@@ -28,13 +28,13 @@ const (
 
 type Handler struct {
 	client kclient.Client
-	log    zerolog.Logger
+	Log    zerolog.Logger
```

Comment:
why do you need to change its visibility ?

Reply:
It is passed to a function in pkg/tls from katalog/main.go file.

Reply:
It is passed to the pkg/tls function from GetServerTLSConfig in  katalog main.go file.